### ISSUE TITLE
Cannot uninstall v19.3.0

### ISSUE THREAD

**Issue by schan** [coreybutler/nvm-windows#910]
nvm version 1.1.10

**Issue:**
Fail to uninstall v19.3.0

**How To Reproduce:**
nvm uninstall 19.3.0

Error message
Uninstalling node v19.3.0...Error removing node v19.3.0
Manually remove C:\bin\nvm\v19.3.0.

**Expected Behavior:**
Uninstalling node v19.3.0... done

**Desktop:**
 - OS: Windows 11

**Comment by coreybutler** [comment]
You're receiving the manual installation notice because there was an error removing the version. 99% of the time, people lack the permissions to remove something from the directory (ex: permissions change on a directory). Every once in a while, npm prevents proper removal. In those cases, manual removal is required to clean up the environment.

Just to illustrate, here's an install/uninstall of that version running as an administrator.

<img width="394" alt="image" src="https://user-images.githubusercontent.com/770982/211052871-a892bea1-1683-431b-9458-0fa5a409eba9.png">

Bottom line, NVM4W resorts to providing that message when something blocks it from doing the uninstallation properly.
